pkgs/data/themes/catppuccin-sddm/default.nix

```diff
@@ -0,0 +1,78 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  just,
+  qtgraphicaleffects,
```

Comment:
Hello, I am the maintainer of catppuccin/sddm. We don't have `qtgraphicaleffects` as a dependency anymore.

Reply:
Thank you ! I'm going to fix this today.